client/src/components/core/Modals/Modals.module.css

```diff
@@ -0,0 +1,55 @@
+
+.modalBtnNewCol {
+    margin: 1rem 0;
+    background-color: aquamarine;
+    border: 5px solid green;
+
+}
+
+.modalDiv {
+    font-family: var(--font-regular);
+}
+
+.modalHeader {
+    font-size: var(--font-size-medium);
+    font-weight: bold;
+}
+
+.modalLabel {
+    font-size: var(--font-size-tiny);
+    color: var(--medium-grey);
+    font-weight: bold;
+}
+
+.modalInput {
+    font-size: var(--font-size-tiny) !important;
+    color: var(--black) !important;
+}
+
+.modalInput::placeholder {
+    color: var(--light);
+}
+
+.boardColumnItemContainer {
```

Comment:
Great job with the names of the classes! this is super helpful when anyone of us needs to come and debug something :)